code-example.html

```diff
@@ -28,10 +28,11 @@
     padding-left: 2em;
     width: 100%;
     overflow: hidden;
+    position: relative;
   }
 
   .source {
-    display: none;
+    display: none !important;
```

Comment:
Whoops, this was not intended I suppose. Now the "show full source" is never visible. I’ll fix it.